experimenter/experimenter/nimbus_ui_new/views.py

```diff
@@ -45,6 +45,23 @@ def get_form_kwargs(self):
         return kwargs
 
 
+class ValidationErrorsMixin:
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        show_errors = self.request.GET.get("show_errors", "") == "true"
+        field_errors = self.get_object().get_invalid_fields_errors()
+
+        validation_errors = {}
+
+        if show_errors:
+            validation_errors = field_errors
+
+        context["validation_errors"] = validation_errors
+
+        return context
```

Comment:
Yep this is perfect, there's also another way to write this like

```py
        show_errors = self.request.GET.get("show_errors", "") == "true"
        field_errors = self.get_object().get_invalid_fields_errors()

        validation_errors = {}
        if show_errors:
            validation_errors = field_errors

        return super().get_context_data(validation_errors=validation_errors, **kwargs)
```

Becuase `get_context_data` turns its `kwargs` into the context.  Just a little simplification.